test_requirements.txt

```diff
@@ -0,0 +1 @@
+pytest
```

Comment:
I think you can remove this file as I included Pytest in the development dependencies in `setup.py`: https://github.com/Farama-Foundation/Jumpy/blob/d79e7aafffa9754a3fc499b7b36bf7d8545f6b82/setup.py#L36=